server/models/classes.py

```diff
@@ -0,0 +1,137 @@
+import requests
+from psycopg2 import Error,  pool
+
+devices_address_range = [
+    'http://localhost:5005',
+    'http://localhost:5004'
+]
+
+
+class Lokals():
+
+    def __init__(self):
+        self.apartments = []
+        self.load_apartment_from_database()
+
+    def create_apartments(self, name):
+        self.apartments.append(Apartment(name))
+
+    def load_apartment_from_database(self):
+        with CursorFromConnectionPool() as cursor:
+            cursor.execute('SELECT apartment_name FROM apartments')
+            apartments_data = cursor.fetchall()
+            for apartment in apartments_data:
+                self.create_apartments(apartment[0])

```

Comment:
The code is saying that you load apartment, but right here you create it. It's ambiguous.